net.goatattack.goatattack.yaml

```diff
@@ -0,0 +1,32 @@
+app-id: net.goatattack.goatattack
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
```

Comment:
```suggestion
runtime-version: '21.08'
```

Reply:
That won't work. It is not C++ 17 ready.

Reply:
well, `cxxflags=-std=c++14`

Reply:
Yay, that works. Thanks!